README.md

```diff
@@ -21,6 +21,20 @@ pip install torchserve torch-model-archiver torch-workflow-archiver
 pip install torchserve-nightly torch-model-archiver-nightly torch-workflow-archiver-nightly
 ```
 
+### 🚀 Quick start with TorchServe (conda)
+
+```
+# Install dependencies
+# cuda is optional
+python ./ts_scripts/install_dependencies.py --cuda=cu111
```

Comment:
PT1.11 the default cuda is 10.2.
cuda 11.1 is gone in PT1.12.

Reply:
So far we've stuck with cuda 10.2. Let's revisit this decision if we choose to move up to 11.

Reply:
Thanks. Updated